airflow/www/templates/airflow/dags.html

```diff
@@ -215,9 +215,9 @@ <h2>DAGs</h2>
 
     </div>
     {% if not hide_paused %}
-    <a href="{{ url_for('admin.index') }}?showPaused=False">Hide Paused DAGs</a>
+    <a onclick="return toggleShowPaused('False')">Hide Paused DAGs</a>
```

Comment:
I don't like using JavaScript for this. Please find another way

Reply:
No problem, I've updated it to use Jinja templating.